src/Organizations/OrganizationsList/hooks/useBuildQuery/useBuildQuery.js

```diff
@@ -0,0 +1,31 @@
+import { useCallback } from 'react';
+
+import {
+  makeQueryBuilder,
+} from '@folio/stripes-acq-components';
+
+import {
+  getKeywordQuery,
+} from '../../OrganizationsListSearchConfig';
+import {
+  customFilterMap,
+  CUSTOM_SORT_MAP,
+} from '../../OrganizationsListFilter/OrganizationsListFilterConfig';
+
+export const useBuildQuery = () => {
+  const buildQuery = useCallback(makeQueryBuilder(
```

Comment:
return useCallback

currently it's code smell

Reply:
updated